src/Symfony/Component/HttpClient/TraceableHttpClient.php

```diff
@@ -51,7 +53,15 @@ public function request(string $method, string $url, array $options = []): Respo
             }
         };
 
-        return $this->client->request($method, $url, $options);
+        $response = $this->client->request($method, $url, $options);
+
+        // Try to convert response to array if possible
+        try {
+            $responseContent = $response->toArray(false);
+        } catch (\Throwable $e) {
```

Comment:
never ever do this: catch throwable and silence any errors
check https://twitter.com/nikita_ppv/status/1042676176836341760 also

Reply:
Thanks for explain with catching.